modelhub/modelhub/models/logistic_regression.py

```diff
@@ -66,8 +67,8 @@ def fit(self, X: 'DataFrame', y: 'SeriesBoolean'):
         if not isinstance(y, SeriesBoolean):
             raise TypeError(f"y is of type {type(y)}, should be SeriesBoolean")
 
-        X_p = X.to_pandas()
-        y_p = y.to_pandas()
+        X_p = X.to_pandas().sort_index()
+        y_p = y.to_pandas().sort_index()
```

Comment:
```suggestion
        data = X.copy()
        data[y.name] = y
        pdf = data.to_pandas()
        X_p = pdf[X.data_columns]
        y_p = pdf[y.name]
```
reasoning: This way you automatically use bach to check if the indexes are the same. Sorting then is not needed anymore, and only one query to export the data.

Reply:
also the docstring should be changed then